snap/snapcraft.yaml

```diff
@@ -4,8 +4,124 @@ summary: OpenStack Exporter for Prometheus
 license: MIT
 contact: solutions-engineering@lists.canonical.com
 description: |
-  The OpenStack exporter, exports Prometheus metrics from a running OpenStack
-  cloud for consumption by prometheus.
+  This snap uses openstack-exporter to provide OpenStack cloud metrics via Prometheus.
+  Grafana dashboards can then be used to visualize the exported metrics, see for example:\
+  https://grafana.com/grafana/dashboards/9701-openstack-dashboard/
+
+  **How-To**
+  ---
+
+  **How to install the snap:**
+
+     sudo snap install charmed-openstack-exporter
+
+     # To allow the snap to read from the /etc/openstack/, you may need to manually add the connection:
+     sudo snap connect charmed-openstack-exporter:etc-openstack
+
+  **How to configure the service:**
+
+  The openstack-exporter service is disabled by default and needs to be configured before the startup.\
+  For example:
+
+     # Set the cloud and os-client-config options
+     sudo snap set charmed-openstack-exporter cloud=mycloud os-client-config=/etc/openstack/clouds.yaml
+
+     # Start the exporter service
+     sudo snap start charmed-openstack-exporter
+
+     # View the logs
+     sudo snap logs charmed-openstack-exporter
+
+  **How to query the metrics:**
+
+  By default the openstack-exporter service serves on localhost:9180/metrics:\
+
+     curl -s http://localhost:9180/metrics
+
+  **How to configure boolean options:**
+
+     # Turn on the option
+     sudo snap set charmed-openstack-exporter multi-cloud=true
+
+     # Turn off the option
+     sudo snap set charmed-openstack-exporter multi-cloud=false
+     # or
+     sudo snap unset charmed-openstack-exporter multi-cloud
```

Comment:
**question (non-blocking):** 

Does it feel necessary to teach users how to configure a boolean value in this section? To me, it seems more fitting for the official documentation on *snapcraft.io*.

Reply:
yes, I agree

Reply:
I think it depends on the implementation. It does not have to be `true` or `false`. It can be something else like yes/no or 0/1. We choose to use these values which are parsed not by snapd but by our configure script